frontend/src/metabase/lib/i18n.js

```diff
@@ -7,36 +7,56 @@ import api from "metabase/lib/api";
 import { DAY_OF_WEEK_OPTIONS } from "metabase/lib/date-time";
 import MetabaseSettings from "metabase/lib/settings";
 
+// we need to be sure to set the initial localization before loading any files
+// so load metabase/services only when we need it
+// load and parse the locale
+const loadMetabaseLocalization = async (locale) => {
```

Comment:
```suggestion
const fetchMetabaseLocalization = async (locale) => {
```

Since this method only fetches the localization object, and the below loadLocalization method actually sets it via `setLocalization`.